pkgs/development/python-modules/jupyter-server-proxy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, notebook
+, simpervisor
+, aiohttp
+, pytest
+, pytest-asyncio
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "jupyter-server-proxy";
+  version = "1.5.3";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "jupyterhub";
+    repo = "jupyter-server-proxy";
+    rev = "v${version}";
+    sha256 = "140mlzcnhadxrnhfwf7glx9cgzrcsbs1gmcsks62a8dpdns89spd";
+  };
+
+  propagatedBuildInputs = [
+    notebook
+    simpervisor
+    aiohttp
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-asyncio
+  ];
+
+  # tests require network
+  doCheck = false;
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.